website/content/partials/datasource/http/Config.mdx

```diff
@@ -0,0 +1,5 @@
+<!-- Code generated from the comments of the Config struct in datasource/http/data.go; DO NOT EDIT MANUALLY -->
```

Comment:
Can't find where this one is coming from. Does the CI fail if you remove it? I think it was generated the first time

Reply:
Let me try to remove it and see what happens

Reply:
Looks like website changes passed with removing this.